### PR TITLE
Fix playlist layout issues.

### PR DESCRIPTION
**Pull request by bkeese**
Fixes layout of playlist share tabs.

### CODE REVIEW

File: app/views/playlists/_share.html.erb [hunk 1]
```diff
@@ -13,7 +13,7 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
-<div id="share-button">
+<div id="share-button" style="margin: -1.6em 1em 1em 0;">
```
Comment (Dananji) [inline app/views/playlists/_share.html.erb]: @bkeese I think it would be better if we could move this inline styling to a CSS file.

Comment (bkeese) [reply app/views/playlists/_share.html.erb]: Agreed. Will you fix please?

File: app/views/playlists/show.html.erb [hunk 2]
```diff
@@ -23,9 +23,13 @@ Unless required by applicable law or agreed to in writing, software distributed
...
 <div class="playlist-view-wrapper row">
   <div class="col-sm-8" id="left_column">
+    <div class="page-title-wrapper playlist-title">
+      <%= icon_only_visibility @playlist.visibility %>
+      <h1 class="page-title"><%= @playlist.title %></h1>
+    </div>
+
     <% if first_playable_item.nil? %>
     <%= image_tag "media-placeholder.png", :class => "img-responsive" %>
-    <%= render 'playlist_title' %>
```
Comment (cjcolvar) [inline app/views/playlists/show.html.erb]: Can we delete `app/views/playlists/_playlist_title.html.erb` then?

Comment (Dananji) [reply app/views/playlists/show.html.erb]: Yes :+1:

**Pull request merged**